doc/generate_logos.py

```diff
@@ -243,9 +243,9 @@ def get_svg_filename_from_versionkey(fn_source, ver):
 
 def searchElementById(node, Id, tagname):
     """
-    Search element by id in the all childs and descendant of node.
+    Search element by id in the all children and descendant of node.
 
-    id is lower case, not ID which is usually used for gelElementById
+    id is lower case, not ID which is usually used for getElementById
```

Comment:
id is supposed to be unique, so the function returns just a single element:
https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById